<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="SKY-20140314HVQ" name="JYWorkTest" tests="1" failures="1" timestamp="16 Apr 2014 10:28:35 GMT" time="0.668" errors="0">
  <testcase name="testWork" time="0.668" classname="JYWorkTest">
    <failure type="org.nutz.ioc.IocException" message="For object [rbf] - type:[null]">
      <![CDATA[org.nutz.ioc.IocException: For object [rbf] - type:[null]
	at org.nutz.ioc.impl.NutIoc.get(NutIoc.java:192)
	at org.nutz.ioc.val.ReferValue.get(ReferValue.java:24)
	at org.nutz.ioc.weaver.FieldInjector.inject(FieldInjector.java:23)
	at org.nutz.ioc.weaver.DefaultWeaver.fill(DefaultWeaver.java:56)
	at org.nutz.ioc.impl.ObjectMakerImpl.make(ObjectMakerImpl.java:96)
	at org.nutz.ioc.impl.NutIoc.get(NutIoc.java:185)
	at org.nutz.ioc.impl.NutIoc.get(NutIoc.java:203)
	at JYWorkTest.testWork(JYWorkTest.java:27)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:702)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:768)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:87)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1188)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1113)
	at org.testng.TestNG.run(TestNG.java:1025)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:109)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:202)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:173)
Caused by: org.nutz.ioc.ObjectLoadException: Wrong type name: 'cn.xuexibao.business.spider.urlfilter.impl.RedisBloomFilter'
	at org.nutz.ioc.IocLoading.E(IocLoading.java:37)
	at org.nutz.ioc.IocLoading.map2iobj(IocLoading.java:64)
	at org.nutz.ioc.loader.map.MapLoader.load(MapLoader.java:86)
	at org.nutz.ioc.loader.cached.CachedIocLoaderImpl.load(CachedIocLoaderImpl.java:47)
	at org.nutz.ioc.impl.NutIoc.get(NutIoc.java:167)
	... 31 more
Caused by: java.lang.ClassNotFoundException: cn.xuexibao.business.spider.urlfilter.impl.RedisBloomFilter
	at java.net.URLClassLoader$1.run(URLClassLoader.java:202)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:190)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:306)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:301)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:247)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:171)
	at org.nutz.lang.Lang.loadClass(Lang.java:1742)
	at org.nutz.ioc.IocLoading.map2iobj(IocLoading.java:60)
	... 34 more
]]>
    </failure>
  </testcase> <!-- testWork -->
</testsuite> <!-- JYWorkTest -->
